tasks/main.yml

```diff
@@ -45,10 +45,11 @@
   apt: name=oracle-java8-unlimited-jce-policy
   when: install_cryptography_extension|bool
 
-- name: Add CA certificate to Java keystore
+- name: Add CA certificates to Java keystore
   java_cert:
-    cert_alias: peopledoc-ca
-    cert_path: /usr/local/share/ca-certificates/devsecops-ca.crt
+    cert_alias: "{{ item.alias }}"
+    cert_path: "{{ item.path }}"
```

Comment:
In order to be generic, `cert_url` could be handled too. Then [`omit`](http://docs.ansible.com/ansible/devel/user_guide/playbooks_filters.html#omitting-parameters) would be required:
```
java_cert:
  cert_alias: item.alias|default(omit) # cert_alias not required when cert_url used
  cert_path: item.path|default(omit)
  cert_url: item.url|default(omit)
```